src/adapter/src/catalog.rs

```diff
@@ -6128,6 +6069,63 @@ impl Catalog {
         Ok(())
     }
 
+    /// Update privileges to reflect the new owner. Based off of PostgreSQL's
+    /// implementation:
+    /// https://github.com/postgres/postgres/blob/43a33ef54e503b61f269d088f2623ba3b9484ad7/src/backend/utils/adt/acl.c#L1078-L1177
+    fn update_privilege_owners(
```

Comment:
There are no changes to the contents of this function. It was just extracted from it's old location so I could write a test on it.